src/SpecialSemanticSignup.php

```diff
@@ -119,7 +119,7 @@ private function userSignup() {
 			$status = $user->sendConfirmationMail();
 
 			if ( !$status->isGood() ) {
-				throw new Exception( wfMsg( 'ses-emailfailed' ) . "\n" . $status->getMessage() );
+				throw new Exception( wfMessage( 'ses-emailfailed' )->escaped() . "\n" . $status->getMessage() );
```

Comment:
Cannot really remember. Probably I thought that this is part of a form "email-field" O_o Thanks for the hint!